code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -90,6 +90,16 @@
 	else
 		return ..()
 
+
+/mob/living/simple_animal/hostile/retaliate/goat/AttackingTarget()
+	. = ..()
+	if(. && ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(istype(H.dna.species, /datum/species/pod))
+			var/obj/item/bodypart/NB = pick(H.bodyparts)
+			H.visible_message("<span class='warning'>[src] takes a big chomp out of [H]!</span>", \
+								  "<span class='userdanger'>[src] takes a big chomp out of your [NB]!</span>")
+			NB.dismember()

```

Comment:
Make sure the code copes if the podperson is a nugget.

And it can't pick the head, right?

Reply:
When you dismember() the chest it drops your internal organs, doesn't drop the bodypart, so it's fine codewise, if a bit insane balance wise.

Andy yeah it can pick the head.